### DISCUSSION TITLE
woof-CE release workflow successful but no releases found

### DISCUSSION THREAD

**Discussion by lakshayrohila** [puppylinux-woof-CE/woof-CE#3418] [Q&A] [answered]
Hi! I have forked woof-CE and tried to make my first puppy build using GitHub actions. The build has completed successfully and according to [https://github.com/puppylinux-woof-CE/woof-CE/wiki/Building-a-Puppy-on-GitHub](url) I must now see the build under **Releases**. But I cannot find the release anywhere. What is the issue?

I am providing the screenshots of what I see-
![screenshot_2022-09-23_1366x768](https://user-images.githubusercontent.com/103126231/192006788-25126334-9d9c-4fe6-b252-d1ec256a1903.png)
![screenshot_2022-09-23_1366x768_000](https://user-images.githubusercontent.com/103126231/192006810-31e4bacf-3b28-4cf5-8e83-f0fa3ebda444.png)

**Comment by dimkr** [comment, marked as answer]
Can you share a screenshot of the releases page? It should be a draft release (not published, only you can see it).

https://github.com/lakshayrohila/woof-CE/releases

**Reply by lakshayrohila** [reply]
Ok, now I got it. Thanks!

I would say that the guide must be updated properly for where to find- under the Releases section on home tab, or under Releases tab.

**Reply by lakshayrohila** [reply]
Ok, so I have edited the page explaining where to find the build if not being found under Releases section after the build has completed.

**Reply by dimkr** [reply]
> Ok, so I have edited the page explaining where to find the build if not being found under Releases section after the build has completed.

Thanks!
